newsfragments/621.misc

```diff
@@ -0,0 +1 @@
+Add PolyBeam for polychromatic experiments
```

Comment:
PolychromaticBeam (also a "i.e. multi-wavelenth / wide bandpass" may also help the novice reader of the change log)